src/meta-srv/src/region/supervisor.rs

```diff
@@ -420,21 +483,25 @@ pub(crate) mod tests {
         let kv_backend = env.kv_backend();
         let peer_lookup = Arc::new(NoopPeerLookupService);
 
-        RegionSupervisor::new(
-            Default::default(),
-            Duration::from_secs(1),
-            selector_context,
-            selector,
-            region_migration_manager,
-            kv_backend,
-            peer_lookup,
+        let (tx, rx) = tokio::sync::mpsc::channel(1024);
```

Comment:
The magic number 1024 appears many times in this PR. I am not sure it's valuable to be configured or made into a constant if we want to change them in the future.